### ISSUE TITLE
return outside function

### ISSUE THREAD

**Issue by Connor69210** [ShadowFient/SupremeBot#1]
it says error return out of function when running and places red on return url also there is syntax error on the colon after (key in form)

**Comment by ShadowFient** [comment]
Yeah, I am aware of it. In fact, I pushed it without compiling it. My local version has solved these problems. After I finish what I want, i will push it.
